src/cpuAV.cpp

```diff
@@ -397,6 +415,46 @@ void CpuAndreVictor::showResponseOnDisplay(string value)
     }
 }
 
+float CpuAndreVictor::getCurrentValue(){
+    if(this->writeIndex){
+        return this->registerOne->getValue();
+    }
+    else(
+        return this->registerTwo->getValue();
+    )
+}
```

Comment:
O `writeIndex` tem que ser comparado a algo.
```suggestion
    if(this->writeIndex == 0){
        return this->registerOne->getValue();
    }
    else(
        return this->registerTwo->getValue();
    )
}
```